tap_github/schemas/workflow_runs.json

```diff
@@ -428,71 +428,63 @@
       "type": "string"
     },
     "head_commit": {
-      "anyOf": [
-        {
-          "type": "null"
+      "type": [
```

Comment:
diff looks larger than it really is because moved from anyOf to just including null in the array of types.  This way is more concise, has the same result.